Added info in the paper about temporat vbc and surface processes (#34)
ggciag/mandyoc: paper/paper.md

Although `Mandyoc` is the acronym for `MANtle DYnamics simulatOr Code`, it is designed to simulate Stokes flow type thermochemical convection taking different compositional layers into account, and it is also appropriate to simulate Earth's lithospheric dynamics in the geological timescale.

> Differing from other codes, `Mandyoc` allows the user to specify spatial and temporal velocity boundary conditions, and the current version incorporates surface processes, simulating erosion and sedimentation.

Reviewer: ```suggestion
`Mandyoc` allows the user to specify spatial and temporal velocity boundary conditions, and the current version incorporates surface processes, imposing rates of erosion and sedimentation on the top of the free surface.
```
